### PR TITLE
[stable/rabbitmq-ha] Fix securityContext for group ID file system per…

### LINKED ISSUE #12025: [stable/rabbitmq-ha] docker-entrypoint.sh: line 196: /var/lib/rabbitmq/.erlang.cookie: Permission denied
**Is this a request for help?**:
NO
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug report

**Version of Helm and Kubernetes**:
Helm 2.12.3
Kubernetes 1.13.3 (on DigitalOcean)

**Which chart**:
stable/rabbitmq-ha

**What happened**:
When installing the chart and using a customized _values.yaml_ file with **persistentVolume** enabled, the first pod loops in in error without ever becoming ready and "Running". By checking the pod's logs one can see the following error:

> $ kubectl -n workspace logs -f rabbitmq-0
> /usr/local/bin/docker-entrypoint.sh: line 196: /var/lib/rabbitmq/.erlang.cookie: Permission denied

**What you expected to happen**:
To have the rabbitmq pods starting and running as usual.

**How to reproduce it** (as minimally and precisely as possible):
`helm install --namespace workspace --name rabbitmq -f rabbitmq-helm-config.yaml stable/rabbitmq-ha`

**Anything else we need to know**:
Our config file worked at the beginning of last week and the previous weeks, we notice that recently the rabbitmq and rabbitmq-ha added a _securityContext_ section that was non-existing a few days ago. Also, we are using this with DigitalOcean to provide dynamic provisioning with Persistent Volume Claims and someone pointed that this error manifests in the stable/rabbitmq for NFS file systems on _stable/rabbitmq_ in #11926

### PR DESCRIPTION
**Pull request by sigmascord**
#### What this PR does / why we need it:
The new `fsGroup` flag allows to enable the group id at a file-system level. 
This is useful to prevent permission errors, especially on clusters with the following Kubernetes cluster flag not set to `RunAsGroup=true`.

Signed-off-by: Samuel Cordeiro <cordeiro.samuel@gmail.com>

#### Which issue this PR fixes
Fixes #12025 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq-ha/README.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ and their default values.
...
 | `replicaCount`                                 | Number of replica                                                                                                                                                                                     | `3`                                                        |
 | `resources`                                    | CPU/Memory resource requests/limits                                                                                                                                                                   | `{}`                                                       |
+| `securityContext.fsGroup`                      | Group ID for the container, mitigates permission errors on a cluster with `RunAsGroup=false`                                                                                                          | `101`                                                      |
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/README.md]: There's no RunAsGroup=false. Can you remove the text `, mitigates permission errors on a cluster with RunAsGroup=false`?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Comment by steven-sheehy** [comment]
Need to also sign your commits. You can run `git commit --amend --signoff && git push -f`

File: stable/rabbitmq-ha/README.md (hunk 1 above)
Comment (steven-sheehy) [reply stable/rabbitmq-ha/README.md]: Oh, you meant the feature gate named that. That actually doesn't impact your problem. You need fsGroup regardless.

Comment (sigmascord) [reply stable/rabbitmq-ha/README.md]: Sure, will do.

Comment (sigmascord) [reply stable/rabbitmq-ha/README.md]: Done. I've changed the field's description to _"Group ID for the container's volumes"_ so one can differentiate it from the `runAsGroup`'s description.

Comment (steven-sheehy) [reply stable/rabbitmq-ha/README.md]: Perfect, thanks

**Comment by steven-sheehy** [comment]
Thanks @sigmascord

**Pull request merged**